app/Http/Controllers/PharmacyController.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Http\Requests\StorePharmacy;
```

Comment:
```suggestion
use App\Http\Requests\CreateUpdatePharmacyRequest;
```
You are also using this request for the update method, so it can't be Store. Also, it is an old but sort of good habit to append -Request to make it clear while reading the code that this is expected to be a request.